src/commands/moderation/repel.ts

```diff
@@ -90,6 +93,39 @@ const checkCanRepelTarget = ({
   return { ok: true };
 };
 
+const sendReasonToTarget = async ({
+  target,
+  reason,
+  guildName,
+  timeoutDuration,
+}: {
+  target: GuildMember | User;
+  reason: string;
+  guildName: string;
+  timeoutDuration?: number;
+}): Promise<boolean> => {
+  const parts: string[] = [];
+  parts.push(`You have been repelled from **${guildName}**.`);
+  if (timeoutDuration && timeoutDuration > 0) {
+    parts.push(`You have been timed out for ${timeToString(timeoutDuration)}.`);
+  }
+  parts.push(`**Reason:** ${reason}`);
+  const messageContent = parts.join('\n');
+
+  const containerComponent = new ContainerBuilder()
+    .setAccentColor(Colors.DarkRed)
+    .addTextDisplayComponents(new TextDisplayBuilder().setContent(messageContent));
+  try {
+    await target.send({
+      flags: MessageFlags.IsComponentsV2,
+      components: [containerComponent],
+    });
+    return true;
+  } catch {
+    return false;
+  }
```

Comment:
personally i would not return anything, and instead throw an error on catch, as well as console.log the error
```suggestion
  try {
    await target.send({
      flags: MessageFlags.IsComponentsV2,
      components: [containerComponent],
    });
  } catch (error) {
    console.log("Unable to send DM to user:\n", error);
    throw new Error(""Unable to send DM to user.");
  }
```
and then ... (see next comment) 1/2